tests/playbooks/tests_bond_options.yml

```diff
@@ -9,201 +9,83 @@
-      import_tasks: tasks/assert_device_present.yml
-      vars:
-        interface: "{{ dhcp_interface2 }}"
-    - name: Test bond options
-      block:
-        - name: "TEST Add Bond with 2 ports"
-          debug:
-            msg: "##################################################"
-        - name: Configure the bond options
-          import_role:
-            name: linux-system-roles.network
-          vars:
-            network_connections:
-              # Create a bond controller
-              - name: "{{ controller_profile }}"
-                state: up
-                type: bond
-                interface_name: "{{ controller_device }}"
-                bond:
-                  mode: 802.3ad
-                  ad_actor_sys_prio: 65535
-                  ad_actor_system: 00:00:5e:00:53:5d
-                  ad_select: stable
-                  ad_user_port_key: 1023
-                  all_ports_active: true
-                  downdelay: 0
-                  lacp_rate: slow
-                  lp_interval: 128
-                  miimon: 110
-                  min_links: 0
-                  num_grat_arp: 64
-                  primary_reselect: better
-                  resend_igmp: 225
-                  updelay: 0
-                  use_carrier: true
-                  xmit_hash_policy: encap2+3
-                ip:
-                  route_metric4: 65535
+        msg: "this is: playbooks/tests_bond_options.yml"
+      tags:
+        - always
 
-              # add an ethernet to the bond
-              - name: "{{ port1_profile }}"
-                state: up
-                type: ethernet
-                interface_name: "{{ dhcp_interface1 }}"
-                controller: "{{ controller_profile }}"
-              # add a second ethernet to the bond
-              - name: "{{ port2_profile }}"
-                state: up
-                type: ethernet
-                interface_name: "{{ dhcp_interface2 }}"
-                controller: "{{ controller_profile }}"
-        - name: Import the task 'assert_device_present.yml'
-          import_tasks: tasks/assert_device_present.yml
-          vars:
-            interface: "{{ controller_device }}"
-        - name: Include the task 'assert_profile_present.yml'
-          include_tasks: tasks/assert_profile_present.yml
+    - name: Test the bond options
+      tags:
+        - tests::bond:create
+      block:
+        - name: Include the task 'run_test.yml'
+          include_tasks: tasks/run_test.yml
           vars:
-            profile: "{{ item }}"
-          loop:
-            - "{{ controller_profile }}"
-            - "{{ port1_profile }}"
-            - "{{ port2_profile }}"
+            lsr_description: Given the system administrators using the network
+              role, when they reconfigure the bond options as specified in
+              `create_bond_profile.yml`, then I can verify that the
+              bond options are configured correctly in sysfs filesystem.
```

Comment:
Actually, the acceptance criteria should describe the steps in `lsr_setup`, `lsr_test` and `lsr_assert`:


A start would be:

Given two DHCP-enabled network interfaces,
When creating a bond profile with them,
Then the controller device and bond port profiles are present and the specified bond options are set for the controller device.